### PR TITLE
Fix & clean up cutout box styling

### LINKED ISSUE #3064: Adjust Header at 768px break point on Primary/Bannered Campaign Pages
This is a follow up ticket to #2935

Tablet view was looking wonky as Youri pointed out. We've decided that it would be best to start the 'mobile' design for primary/bannered campaign pages earlier at the 768px break point. This design makes it easier for the user's eye to go from the header to the body text.

## List of Requirements
- [ ] Start mobile design (12 column header) at 768px breakpoint

Redpen here (first two designs): https://redpen.io/p/ys4594fedace293c98

### Old Design:

Screen Shot 2019-04-26 at 4 45 59 PM

### New Design:

Screen Shot 2019-04-26 at 4 39 04 PM

### PR DESCRIPTION
**Pull request by mmmavis**
Closes #3064

### main site
- Primary page: https://foundation-mofostaging-pr-3456.herokuapp.com/en/about/
- Bannered Campaign page: https://foundation-mofostaging-pr-3456.herokuapp.com/en/campaigns/bannered-campaign-page/

### MozFest (Need to change setting on review app for these pages to show. See instructions below)

- MozFest Home: https://foundation-mofostaging-pr-3456.herokuapp.com/en/mozilla-festival/
- other pages: https://foundation-mofostaging-pr-3456.herokuapp.com/en/mozilla-festival/spaces/

### CODE REVIEW

**Comment by mmmavis** [comment]
@youriwims it's easier to review this PR without whitespace changes: https://github.com/mozilla/foundation.mozilla.org/pull/3456/files

**Comment by beccaklam** [comment]
@mmmavis The MozFest pages look good but I had trouble going to the foundation site homepage sites. I tried to follow your instructions but ended up with a server 500 error?

File: source/sass/global.scss [hunk 1]
```diff
@@ -40,20 +40,27 @@ button:hover {
...
     @media screen and (min-width: $bp-md) {
-      $offset-left: 24px;
+      margin-left: -$offset-x-md;
+      margin-right: -$offset-x-md;
+    }
 
-      margin-left: -$offset-left;
-      padding-left: $offset-left;
+    .cutout {
+      $grid-gutter-width: map-get($grid-gutter-widths, xs);
```
Comment (youriwims) [inline source/sass/global.scss]: I like this map-get function a lot 🙌  but, where's `$grid-gutter-widths` defined?

File: source/sass/mozfest.scss [hunk 2]
```diff
@@ -39,23 +39,11 @@ body.mozfest {
         }
       }
 
-      .full-width-cutout {
-        $grid-gutter-width: map-get($grid-gutter-widths, xs);
-        $padding-x: #{$grid-gutter-width / 2};
-
+      .cutout {
+        z-index: $banner-background-z-index + 2;
```
Comment (youriwims) [inline source/sass/mozfest.scss]: It seems like we only need this z-index starting at medium breakpoint. What do you think @mmmavis or it doesn't really matter?

**Comment by youriwims** [comment]
> @youriwims it's easier to review this PR without whitespace changes: https://github.com/mozilla/foundation.mozilla.org/pull/3456/files

btw @mmmavis ☝️ url didn't remove the whitespace, I still had to manually check that box in the diff settings. _Butttttttt_ if you add `?w=1` on the end of the url, you can view the diff without whitespace 😄

**Comment by mmmavis** [comment]
@beccaklam omg I just learned that you can actually see MozFest pages without going through complicated set up...

Just click on "Mozilla Festival" from the main site's nav then you should be able to navigate the MozFest (ignore the URLs though. We have to include `/mozilla-festival` in order for MozFest site to show on review app)

image

Check out the PR description again for links to 👀

**Comment by mmmavis** [comment]
@youriwims lol yea I meant to include `w=1` in this [comment](https://github.com/mozilla/foundation.mozilla.org/pull/3456#issuecomment-515583464) but forgot to do so before I hit `comment` 🤦‍♂

**Comment by beccaklam** [comment]
> @beccaklam omg I just learned that you can actually see MozFest pages without going through complicated set up...
> 
> Just click on "Mozilla Festival" from the main site's nav then you should be able to navigate the MozFest site. Ignore the page URLs though (e.g., https://foundation-mofostaging-pr-3456.herokuapp.com/en/mozilla-festival/spaces/) . We have to include `/mozilla-festival` in order for MozFest site to show on review app.
> 
> image
> 
> Check out the PR description again for links to 👀

Haha thanks. I'll do that next time, Youri already did some heroku rebuilding so I could QA the main site's pages. It's good to go from a design POV!

File: source/sass/global.scss (hunk 1 above)
Comment (mmmavis) [reply source/sass/global.scss]: It's from `/node_modules/bootstrap/scss/_variables.scss` (which get imported into `main.scss`)

File: source/sass/mozfest.scss (hunk 2 above)
Comment (mmmavis) [reply source/sass/mozfest.scss]: Ummm I think we can keep it there. If we do this just for medium+ then we'll have to adjust a bunch of other elements' `z-index` for different screen sizes which might make the code less readable.

**Comment by mmmavis** [comment]
@youriwims this is ready for code review again!

File: source/sass/mozfest.scss (hunk 2 above)
Comment (youriwims) [reply source/sass/mozfest.scss]: cool cool 👍

**Pull request merged**